app/analytics.go

```diff
@@ -182,14 +182,16 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			return rows, nil
 		}
 
-		return a.Srv.Store.Post().AnalyticsPostCountsByDay(teamId, true, false)
+		postCountsOptions := &model.AnalyticsPostCountsOptions{TeamId: teamId, BotsOnly: true, YesterdayOnly: false}
```

Comment:
nit: I think it would be fine to inline this as:
```go
return a.Srv.Store.Post().AnalyticsPostCountsByDay(&model.AnalyticsPostCountsOptions{
    TeamId: teamId, 
    BotsOnly: true, 
    YesterdayOnly: false,
})
```

Reply:
Got it.  Thanks for the feedback, @lieut-data.  One more commit coming.